src/api/config/routes.rb

```diff
@@ -243,9 +243,9 @@ def self.public_or_about_path?(request)
 
     controller 'webui/project' do
       get 'project/' => :index, as: 'projects'
-      get 'project/list_public' => :index
-      get 'project/list_all' => :index, show_all: true
-      get 'project/list' => :index
+      get 'project/list_public' => :index, as: 'project_list_public'
```

Comment:
[nitpick] I am not really a fan of giving everything a name :unamused:

Reply:
> [nitpick] I am not really a fan of giving everything a name unamused

So how do you propose to reference this routes in the breadcrumb then?

Reply:
Yes